src/core/Akka/Actor/Scheduler/TimerScheduler.cs

```diff
@@ -41,36 +41,39 @@ public Timer(object key, object msg, bool repeat, int generation, ICancelable ta
             }
         }
 
-        public interface ITimerMsg
+        public interface ITimerMsg : IWrappedMessage, INoSerializationVerificationNeeded
```

Comment:
`IWrappedMessage` is how we can get access to the contents of the timer and record it via Phobos metrics.